en/docs/integrate/api-led-integration.md

```diff
@@ -0,0 +1,56 @@
+# API-led Integration
+
+WSO2 API Manager 4.0.0 consists of an API management layer as well as an integration layer. While the integration layer (Micro Integrator) is used for running the integration logic, the API management layer is used for managing the APIs (with the integration logic) and making these APIs discoverable to developers.
```

Comment:
Let's exclude version number or we will have to remember to manually update this for the next release if there is one.